src/patterns/components/spinners/large.hbs

```diff
@@ -0,0 +1,8 @@
+---
+name: Large Spinner
+notes: The large variant is used in special cases.
```

Comment:
Do we have any example's of special cases that would could detail out?

Reply:
@philrinke thoughts?